docs/installation/manual/installing_karaf_and_alpaca.md

```diff
@@ -0,0 +1,346 @@
+# You may want to wait a bit for Karaf to start.
+# If you're not sure whether or not it's running, you can always run:
+# ps aux | grep karaf
+# to see if the server is up and running.
+/opt/karaf/bin/client feature:install wrapper
+/opt/karaf/bin/client wrapper:install
+/opt/karaf/bin/stop
+```
+
+### Creating and Starting the Karaf Service
+
+Installing the Karaf wrapper generates several service files that can be used on different types of systems. For this example installation on an Ubuntu 18.04 machine, we want to enable the `karaf.service` service so that Karaf is properly started on boot.
+
+```bash
+sudo systemctl enable /opt/karaf/bin/karaf.service
+sudo systemctl start karaf
+```
+
+## Alpaca 1.0.x
+
+### Adding the Required Karaf Repositories
+
+Karaf features can be installed from several different types of sources, but the fastest and easiest way to do so is from existing repository URLs that we can just plug into Karaf to provide us feature lists prepared and ready for installation. Like most interactions with Karaf, we can add these repositories using its built-in `client`.
+
+!!! notice
+    These repositories are updated consistently, and their updates include revised dependency lists. Commonly, when repositories are out of date or otherwise mismatched, feature installation can result in an `Unable to resolve root: missing requirement` error; for this reason, this guide recommends using recently-updated versions of these repositories. That being said, if such errors occur despite installing the latest versions of these features, the maintainer of the features repository should be informed.
+
+For the Karaf features we’re going to install, we need a few different repositories to be added to the list:
+
+```bash
+/opt/karaf/bin/client repo-add mvn:org.apache.activemq/activemq-karaf/ACTIVEMQ_KARAF_VERSION/xml/features
+/opt/karaf/bin/client repo-add mvn:org.apache.camel.karaf/apache-camel/APACHE_CAMEL_VERSION/xml/features
+/opt/karaf/bin/client repo-add mvn:ca.islandora.alpaca/islandora-karaf/1.0.1/xml/features
+# XXX: This shouldn't be strictly necessary, but appears to be a missing
+# upstream dependency for some fcrepo features.
+/opt/karaf/bin/client repo-add mvn:org.apache.jena/jena-osgi-features/3.1.1/xml/features
+```
+- `ACTIVEMQ_KARAF_VERSION`: The most recent version of `activemq-karaf` 5.x.x you can find on the [activemq-karaf Maven repository page](https://mvnrepository.com/artifact/org.apache.activemq/activemq-karaf).
+- `APACHE_CAMEL_VERSION`: The most recent version of `apache-camel` 2.x.x you can find on the [apache-camel Maven repository page](https://mvnrepository.com/artifact/org.apache.camel.karaf/apache-camel).
+
+### Installing the Required Karaf Features
+
+Before we can configure the features we’re going to use, they need to be installed. Some of these installations may take some time.
+
+```bash
+/opt/karaf/bin/client feature:install fcrepo-service-activemq
+/opt/karaf/bin/client feature:install jena
```

Comment:
Jena doesn't appear to be working. Karaf responds with: ```
vagrant@claw:/opt$ /opt/karaf/bin/client repo-add mvn:org.apache.jena/jena-osgi-features/3.1.1/xml/features
Logging in as karaf
Adding feature url mvn:org.apache.jena/jena-osgi-features/3.1.1/xml/features
vagrant@claw:/opt$ /opt/karaf/bin/client feature:install fcrepo-service-activemq
Logging in as karaf
vagrant@claw:/opt$ /opt/karaf/bin/client feature:install jena
Logging in as karaf
Error executing command: Error:
	Error downloading mvn:org.apache.jena/jena-osgi/3.1.0-SNAPSHOT
vagrant@claw:/opt$
```

Besides, aren't we using blazegraph instead of Jena?

Reply:
Upgrading to `repo-add mvn:org.apache.jena/jena-osgi-features/3.13.1/xml/features` seems to be working.